erpnext/shopping_cart/doctype/shopping_cart_settings/shopping_cart_settings.js

```diff
@@ -1,31 +1,37 @@
 // Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
 // License: GNU General Public License v3. See license.txt
 
-$.extend(cur_frm.cscript, {
-	onload: function() {
-		if(cur_frm.doc.__onload && cur_frm.doc.__onload.quotation_series) {
-			cur_frm.fields_dict.quotation_series.df.options = cur_frm.doc.__onload.quotation_series;
-			cur_frm.refresh_field("quotation_series");
+frappe.ui.form.on("Shopping Cart Settings", {
+	onload: function(frm) {
+		if(frm.doc.__onload && frm.doc.__onload.quotation_series) {
+			frm.fields_dict.quotation_series.df.options = frm.doc.__onload.quotation_series;
+			frm.refresh_field("quotation_series");
 		}
 	},
-	refresh: function(){
-		toggle_mandatory(cur_frm)
+	refresh: function(frm){
+		toggle_mandatory(frm)
 	},
-	enable_checkout: function(){
-		toggle_mandatory(cur_frm)
+	enable_checkout: function(frm){
+		toggle_mandatory(frm)
 	},
-	enabled: function() {
-		if (cur_frm.doc.enabled === 1) {
-			cur_frm.doc.show_configure_button = 1;
-			cur_frm.refresh_field('show_configure_button');
+	enabled: function(frm) {
+		if (frm.doc.enabled === 1) {
+			frm.doc.enable_variants = 1;
+			frm.refresh_field('enable_variants');
+			frm.toggle_reqd("company", true);
+			frm.toggle_reqd("default_customer_group", true);
```

Comment:
Instead of if else, use below code

```
let is_required = frm.doc.enabled ? 1:0;
frm.toggle_reqd(["company", "default_customer_group"], is_required);
```